os/task-scheduler-with-systemd-timers.md

```diff
@@ -0,0 +1,63 @@
+# Task scheduler with systemd timers
+
+CoreOS doesn't use `cron` scheduling daemon. The replacement is `systemd` timer unit. Here we will show you how you can schedule a periodic job.
```

Comment:
I feel like this leads off with a negative. Should we say "CoreOS uses systemd timers to..." and then say that its similar to cron, but with better syntax.